<?xml version="1.0"?>
<launch>
  <arg name="cubemarker_size" default="0.06" />
  <arg name="reserve_area" default="0.02" />
  <arg name="source_frame" default="/photoneo_center_optical_frame" />
  <arg name="target_frame" default="/base_link" />

  <node pkg="oneshot_calibration_system" type="ar_cube_pointcloud_extractor" name="ar_cube_pointcloud_extractor" output="screen">
    <param name="cubemarker_size" value="$(arg cubemarker_size)" />
    <param name="reserve_area" value="$(arg reserve_area)" />
    <param name="target_frame" value="$(arg target_frame)" />
    <param name="source_frame" value="$(arg source_frame)" />

    <rosparam file="$(find oneshot_calibration_system)/config/oneshot_calibration_config.yaml" command="load" />

    <remap from="input_pointcloud" to="/photoneo_center/pointcloud" />
    <remap from="output_pointcloud" to="/extracted_pointcloud" />

  </node>
</launch>
